docs/content/en/docs/reference/support/support-versions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Version support"
+linkTitle: "Version support"
+weight: 50
+description: >
+  EKS Anywhere and Kubernetes version support policy
+---
+
+To see supported versions of Kubernetes for each release of EKS Anywhere, and information about that support, refer to the content below.
+
+
+## Kubernetes support
+
+Each EKS Anywhere version generally includes support for multiple Kubernetes versions, with the exception of the initial few releases.
+Starting from EKS Anywhere version 0.11, the latest version supports at least four recent versions of Kubernetes.
+The end of support date of a Kubernetes version aligns with Amazon EKS in AWS as documented on the [Amazon EKS Kubernetes release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar). 
+
+Common vulnerabilities and exposures (CVE) patches and bug fixes, including those for the supported Kubernetes versions, are back-ported to the latest EKS Anywhere version (version n) and the previous one version (version n-1).
+The following table shows EKS Anywhere version support for different Kubernetes versions:
+
+| Kubernetes version      | Supported EKS Anywhere version | First supported | End of support  |
+|------------|---------------------|---------------------------------|-------------------------|
+| 1.23 | 0.11 | August 18, 2022	 | October 2023 | 
+| 1.22 | 0.11, 0.10, 0.9, 0.8 | March 31, 2022 | May 2023 | 
+| 1.21 | 0.11, 0.10, 0.9, 0.8, 0.7, 0.6, 0.5 | September 8, 2021 | February 2023 | 
+| 1.20 | 0.11, 0.10, 0.9, 0.8, 0.7, 0.6, 0.5 | September 8, 2021 | November 1, 2022 | 
+| 1.19 | Not supported | -- | -- | 
+| 1.18 | Not supported | -- | -- | 
+
+The following table notes which EKS Anywhere and related Kubernetes versions are currently supported for CVE patches and bug fixes:
+
+| EKS Anywhere version      | Kubernetes versions included | EKS Anywhere Release Date | CVE patches and bug fixes back-ported? |
+|------------|---------------------|---------------------------------|-------------------------|
+| 0.11 | 1.23, 1.22, 1.21, 1.20 | August 18, 2022 | Yes | 
+| 0.10 | 1.22, 1.21, 1.20 | June 30, 2022 | Yes | 
+| 0.9 | 1.21, 1.20 | May 12, 2022 | No | 
+| 0.8 | 1.22, 1.21, 1.20 | March 31, 2022 | No | 
+| 0.7 | 1.21, 1.20 | January 27, 2022 | No | 
+| 0.6 | 1.21, 1.20 | October 29, 2021 | No | 
+| 0.5 | 1.21, 1.20 | September 8, 2021 | No | 
```

Comment:
what is our support commitment on these old releases?

Reply:
TBD. Jackson and I are still thinking about it. We don't have to make any public statements yet.